internal/signer/linux/util/util_test.go

```diff
@@ -0,0 +1,35 @@
+package util
+
+import (
+	"testing"
+)
+
+func TestLoadConfig(t *testing.T) {
```

Comment:
can you add a negative test case as well

Reply:
Done~